code/style.css

```diff
@@ -1,7 +1,260 @@
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  background-color: #cce0ff;
+  margin: 0;
+  font-family: 'Oswald', sans-serif;
+}
+
+
+/* Mobile first approach */
+/* Header */
+header {
+  height: 300px;
+  width: 100%;
+  margin: 0 0 20px 0;
+  display: flex;
+  align-items: center; 
+}
+
+
+.navigation {
+  width: 100%;
+  height: auto;
+  position: absolute;
+  left: 0px;
+  top: 0px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  font-size: 20px;
+  position: fixed;
+  background-color: blanchedalmond;
+  opacity: 0.5;
+}
+
+
+a {
+  text-decoration: none;
+  color: darkgreen;
+  font-weight: bold;
+}
+
+
+.hero-image {
+  width: 100%;
+  height: 100%;
+  background-image: url('./making-bouquet.jpg');
+  background-position: center;
+  background-size: cover;
+}
+
+.hero-text {
+  position: absolute;
+  padding: 20px;  
+  margin: 20px auto;
+}
+
+
+h1 {
+  font-family: 'Dancing Script', cursive;
+  font-size: 30px;
+}
+
+
+/* Main section */
+.main-section {
+  display: grid;
+  grid-template-columns: 1fr;  
+}
+
+/* Sign up form */
+h2 {
+  text-align: center;
+}
+
+
+.form {
+  height: auto;
+  width: 90%;
+  margin: 10px auto;
+  padding: 10px;
+  border: 5px solid blanchedalmond;
+  border-radius: 10px;
+  font-family: sans-serif;
+  color: whitesmoke;
+  background-color: lightgrey;
+}
+
+
+label {
+  display: block;
+  padding: 5px;
+}
+
+
+input {
+  display: block;
+  border: none;
+}
+
+
+input:valid {
+  background-color: #ccffd1;
+}
+
+
+/* Order form */
+h3 {
+  text-align: center;
+}
+
+.flowers {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  border: none;
+}
+
+
+.flower-image {
+  height: auto;
+  width: 90%;
+  object-fit: cover;
+  border-radius: 15px;
+  cursor: pointer;
+}
+
+
+
+/* Video and accordion */
+.video-faq {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+
+.video {
+  object-fit: cover;
+  height: auto;
+  width: 95%;
+  margin: 10px auto;
+  border: 1px solid black;
+  border-radius: 15px;
+}
+
+
+.accordion {
+  background-color: blanchedalmond;
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  height: auto;
+  margin: 10px auto;
+  padding: 5px;
+  border-radius: 10px;
+  border: 5px solid lightgrey;
+  cursor: pointer;
+}
+
+
+.question {
+  height: auto;
+  text-align: center;
+}
+
+
+.question::after {
+  font-family: "Font Awesome 5 Free";
+  content: '\f078';
+  font-weight: bold;
+}
+
+
+.answer {
+  display: none;
+  background-color: whitesmoke;
+  color: black;  
+}
+
+
+p {
+  padding: 5px;
+}
+
 
 .active + .answer {
-  /* some styling here */
+  display: block; 
+}
+
+.active::after {
+  content: '\f077';
+}
+
+
+/* Media queries */
+@media (min-width: 768px) {
+  header {
+    height: 500px;
+  }
+
+
+  h1 {
+    font-size: 60px;
+  }
+
+
+  .main-section {
```

Comment:
I love the grid, and how easy it is to change the content into more columns, nice use!